From 14f665d63fe7b62cc9c3f1e3afb68d82bf1914d9 Mon Sep 17 00:00:00 2001
From: Eric Steinborn <esteinborn@gmail.com>
Date: Tue, 19 Aug 2014 17:09:31 -0400
Subject: [PATCH 050/214] Update to Universal Google Analytics Script

Google is upgrading to their new "Universal Analytics."

I'd recommend building that into this instead of the legacy ga.js script.
https://developers.google.com/analytics/devguides/collection/analyticsjs/
---
 _includes/analytics.html | 23 ++++++++++-------------
 1 file changed, 10 insertions(+), 13 deletions(-)

diff --git a/_includes/analytics.html b/_includes/analytics.html
index 21684d7..d8cdc0e 100644
--- a/_includes/analytics.html
+++ b/_includes/analytics.html
@@ -1,16 +1,13 @@
 {% if site.google_analytics %}
 	<!-- Google Analytics -->
-	<script type="text/javascript">
-
-	  var _gaq = _gaq || [];
-	  _gaq.push(['_setAccount', '{{ site.google_analytics }}']);
-	  _gaq.push(['_trackPageview']);
-
-	  (function() {
-	    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
-	    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
-	    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
-	  })();
-
+	<script>
+		(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+		(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+		m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+		})(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+		
+		ga('create', '{{ site.google_analytics }}', 'auto');
+		ga('send', 'pageview');
 	</script>
-{% endif %}
\ No newline at end of file
+	<!-- End Google Analytics -->
+{% endif %}
-- 
2.17.1

